data/json/recipes/recipe_food.json

```diff
@@ -5879,8 +5879,7 @@
     "qualities": [ { "id": "CUT", "level": 1 }, { "id": "COOK", "level": 1 } ],
     "tools": [ [ [ "char_smoker", 10 ] ] ],
     "components": [
-      [ [ "offal", 4 ], [ "meat_nofish", 2, "LIST" ], [ "brain", 10 ] ],
-      [ [ "stomach", 2 ], [ "stomach_large", 1 ] ],
+      [ "sausage_wasteland_raw", 1 ],
       [
         [ "salt_water", 4 ],
```

Comment:
I don't think these are still needed. And you may want to keep the original recipe too, in case people find the extra step from ingredients to raw sausage and then to cooked sausage tedious.